plugins/services/src/js/routes/services.js

```diff
@@ -171,44 +180,117 @@ const serviceRoutes = [
           },
           {
             type: Route,
+            path: "configuration",
+            title: "Configuration",
+            component: HighOrderServiceConfiguration,
+            buildBreadCrumb() {
```

Comment:
We actually don't use the `#buildBreadCrumb` method anymore, right? As a separate task, we should remove all of these methods, but let's not add any more.